horace_core/sqw/coord_transform/@line_proj/private/transform_img_to_hkl_.m

```diff
@@ -17,7 +17,7 @@
 else
     bmat = obj.bmatrix();
 end
-hkl_to_img = bmat*q_to_img;
+hkl_to_img = q_to_img*bmat;
 offset = bmat\offset_cc;
```

Comment:
I would make a ticket and mark a concern here for further investigation (for me and Toby). 
In `aProjectionBase` class, rows 624-647 is a generic version of transform_img_to_hkl function. 

There is a generic transformation from img to pix and then transformation from pix to hkl. This defines the order of matrix operation which was there before and have been changed by the commit. 

I do not understand the reason why inverse order gives correct answer.

Reply:
Ticket #1318 created to track.